app/src/renderer/js/main.js

```diff
@@ -14,6 +15,8 @@ const {app} = remote;
 // Observers that should be disposed when the window unloads
 const observersToDispose = [];
 
+const cropperWindowBuffer = 2;
```

Comment:
It's not clear enough from the name what this is and why it's 2. Needs a succinct code comment.

Reply:
I got that from [here](https://github.com/wulkano/kap/blob/master/app/src/main/index.js#L32). I'm not 100% sure what its purpose is. I'm guessing to make up for the 1px of the border of the cropper window. Is that correct?

Reply:
It was originally added in https://github.com/wulkano/kap/commit/13a0cd6d66666cd2f5f395ebf001a1de4ced50d1

Reply:
ed705f7